lib/scihist_digicoll/oh_microsite_import_utilities.rb

```diff
@@ -0,0 +1,293 @@
+# Utility methods for oral history microsite import.
+# See:
+# scihist:oh_microsite_import:import_interviewee_biographical_metadata
+
+require "shrine"
+
+module OhMicrositeImportUtilities
+  module Updaters
+
+    def self.birth_date(oral_history_content, rows)
+      oral_history_content.interviewee_birth ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_birth.date = keep_yyyy_mm_dd(rows.first['birth_date'])
+    end
+    def self.birth_city(oral_history_content, rows)
+      oral_history_content.interviewee_birth ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_birth.city         = rows.first['birth_city']
+    end
+    def self.birth_state(oral_history_content, rows)
+      oral_history_content.interviewee_birth ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_birth.state        = rows.first['birth_state']
+    end
+    def self.birth_province(oral_history_content, rows)
+      oral_history_content.interviewee_birth ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_birth.province     = rows.first['birth_province']
+    end
+    def self.birth_country(oral_history_content, rows)
+      oral_history_content.interviewee_birth ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_birth.country      = rows.first['birth_country']
+    end
+
+    def self.death_date(oral_history_content, rows)
+      oral_history_content.interviewee_death ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_death.date = keep_yyyy_mm_dd(rows.first['death_date'])
+    end
+    def self.death_city(oral_history_content, rows)
+      oral_history_content.interviewee_death ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_death.city         = rows.first['death_city']
+    end
+    def self.death_state(oral_history_content, rows)
+      oral_history_content.interviewee_death ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_death.state        = rows.first['death_state']
+    end
+    def self.death_province(oral_history_content, rows)
+      oral_history_content.interviewee_death ||= OralHistoryContent::DateAndPlace.new
+      oral_history_content.interviewee_death.province     = rows.first['death_province']
+    end
```

Comment:
put a blank line between method defs makes it easier to read and is more conventional, thanks! (that is empty line after `end`)

Reply:
Done.